cmd/tinygotest/validate_schema_test.go

```diff
@@ -0,0 +1,34 @@
+//go:build tinygo
+// +build tinygo
+
+package main
+
+import (
+	"encoding/xml"
+	"fmt"
+	"testing"
+)
+
+// TestXMLValidation tests XML validation
```

Comment:
@cognitivegears shall we drop this?

Reply:
Thanks sorry I missed this one, will remove

Reply:
Done